app/services/stay/list_child_costs.rb

```diff
@@ -46,7 +46,7 @@ def child_cost(child)
     if child.age >= 5
       stay.needs_bed? ? :child : :child_meal
     else
-      stay.needs_bed? ? :infant : nil
+      stay.needs_bed? ? :infant : :child_meal
```

Comment:
I'm not too familiar with the business rules here, but it looks like we're changing the cost from, I guess, no cost (nil) to something that has a cost (child_meal). I just wanted to make sure that this change is inline with desired business rules. What do you think?

Reply:
Good call on this.
Confirmed with client that it is supposed to be nil, thanks so much 🎖